scripts/post_install.js

```diff
@@ -11,9 +11,10 @@ const tar = require('tar')
 const pkg = require('../package.json')
 
 const name = `${os.platform()}-${os.arch()}`
+const buildFromSource = process.env.npm_config_build_from_source
 
 if (process.env.DD_NATIVE_METRICS !== 'false') {
-  if (process.env.DD_SKIP_PREBUILT_DOWNLOAD !== 'true') {
```

Comment:
should we keep this for backwards compatibility for anyone already using it?

Reply:
This has not been released yet.

Reply:
well then... I suppose that is what I get for not keeping as up to date on things... :(